libplatsupport/CMakeLists.txt

```diff
@@ -42,6 +42,18 @@ file(
         src/plat/${KernelPlatform}/acpi/*.c
 )
 
+if(${KernelArch} STREQUAL "arm")
```

Comment:
Why add these in addition to the file(GLOB ..) call above?

Reply:
because I should have also deleted the glob here. Updated. I don't want to compile in the generic_ltimer if its not selected for the platform specifically. On my branch I have 

```
if(KernelPlatformSDM845)
    list(APPEND deps src/arch/arm/generic_ltimer.c)
endif()
```

Reply:
FYI the commit message in 41f0b2a8af9393fec655a05cd7d9a5d1caf01504 says you always compile the generic_timer.

Reply:
Ah, one refers to generic_timer.c and the other to generic_ltimer.c.  I understand now

Reply:
Do you want to add `  list(APPEND deps src/arch/arm/generic_timer.c) ` here so that it is the same as it was before?

Reply:
Done